src/main/java/pl/krakow/phonebook/customer/controller/CustomerController.java

```diff
@@ -0,0 +1,51 @@
+package pl.krakow.phonebook.customer.controller;
```

Comment:
So, you put it like <Resource>/<Layers>. Usually it is done in another way: <Layer>/<Resources> e.g.
`.controllers.CustomerController`
`.services.CustomerService`
`.repo.`customerRepository` etc.

That is not a big thing here but in a real project that would be a source of tensions.